processor/configuration/src/main/resources/schemas/actions/kafka_topic_sink_0.1.json

```diff
@@ -7,9 +7,30 @@
       "title": "Topic Name",
       "description": "The topic where to send the event.",
       "example": "my-topic"
+    },
+    "kafka_broker_url": {
+      "type": "string",
+      "title": "Broker URL",
+      "description": "RHOSAK Broker URL ",
+      "example": "username-c--hj---mhlksdfss-p--a.bf2.kafka.rhcloud.com:443"
```

Comment:
I'm not sure about providing this as an example. Is it some existing instance?

Reply:
it's an obfuscated string. As RHOSAK URLs resemble this, why not?